prisma/datamodel.prisma

```diff
@@ -2,12 +2,31 @@ type User {
   id: ID! @unique
   email: String! @unique
   password: String!
-  name: String
+  displayName: String
   photoUrl: String
-  description: String
-  createdAt: DateTime!
-  updatedAt: DateTime!
+  statusMsg: String
   friends: [User!]! @relation(name: "Friend")
+  isOnline: Boolean
+  created: DateTime!
+  updated: DateTime!
+}
+
+type Chatroom {
+  id: ID! @unique
+  lastChat: Chat @relation(name: "LastChat")
```

Comment:
I can't understand this field, what's mean of `lastChat` and `lastChatCnt` in Chatroom?